buildout.cfg

```diff
@@ -26,7 +26,10 @@ eggs =
 # development egg, you really need to un-pin it here.
 lizard-auth-server =
 Django = 1.6.6
+PyJWT = 1.4.0
```

Comment:
Is deze versie (en die van fake-factory en future) echt specifiek nodig? Anders kan 'ie beter in de lijst "reported by buildout".